.env.test

```diff
@@ -0,0 +1,11 @@
+# Flask server configuration.
+FLASK_APP=app
+FLASK_ENV=development
+FLASK_SKIP_DOTENV=True
+
+# Trello Config
+TRELLO_BASE_URL=https://api.trello.com/1
```

Comment:
Quick point that should have actually been covered in module 1 or 2 - your README doesn't explain how to get the project up and running.

The [starter repo](https://github.com/CorndelWithSoftwire/DevOps-Course-Starter) has actually been updated since you started the course to have an extra section at the bottom about env variables.
Importantly, you should update the README to explain any important environment variables that other developers would need to set to run the project and .env.example should be kept up to date as a point of reference for them.

Reply:
Iv'e added instructions to the README.md and added the environment variables to .env